wled00/wled_server.cpp

```diff
@@ -15,8 +15,9 @@
  * Integrated HTTP web server page declarations
  */
 
-bool handleIfNoneMatchCacheHeader(AsyncWebServerRequest* request);
-void setStaticContentCacheHeaders(AsyncWebServerResponse *response);
+bool handleIfNoneMatchCacheHeader(AsyncWebServerRequest* request, int code, const String &eTagSuffix = "");
```

Comment:
Could you avoid String?

Reply:
Sure, we could also use `uint16_t`. I'll change it later.

Reply:
I know that String has some advantages but if used improperly it may fragment memory.
Do whatever you want.